cmd/mycelo/main.go

```diff
@@ -469,5 +475,6 @@ func loadBot(ctx *cli.Context) error {
 		TransactionsPerSecond: ctx.Int("tps"),
 		Clients:               clients,
 		Verbose:               verbose,
+		MaxPending:            ctx.Uint64("maxpending"),
```

Comment:
we should start using `loadTestMaxPendingFlag.Name` instead of a string

Reply:
Switched this and `loadTestTPSFlag` as well